README.md

```diff
@@ -22,6 +22,10 @@
 
 [View the docs here](https://reacttraining.com/react-router)
 
+[3.x docs](https://github.com/ReactTraining/react-router/blob/v3.0.2/docs)
```

Comment:
Instead of specifically the `3.0.2` tag, this should be the `v3` branch so we don't have to updated the link every time v3's version gets bumped.